docs/RDMA-example.md

```diff
@@ -0,0 +1,69 @@
+
+# RDMA example
+
+This guide provides example commands that can be used to send a video file via RDMA on a single node.
+
+## Prerequisites
+
+1. Media Communications Mesh is built
+```
+./build.sh
+```
+
+2. Libfabric is build and installed with RDMA capable providers
```

Comment:
is installed already?

Maybe:
"Libfabric should be build already and installed with RDMA capable providers."

Reply:
changed